Python/Product/PythonTools/PythonTools/Options/PythonInterpreterOptionsControl.cs

```diff
@@ -60,6 +60,10 @@ public PythonInterpreterOptionsControl()
                 _defaultInterpreter.Items.Clear();
 
                 foreach (var interpreter in _serviceProvider.GetPythonToolsService().InterpreterOptions.Select(x => x.Key).OrderBy(f => f.Description)) {
+                    if (!Microsoft.PythonTools.Interpreter.PythonInterpreterFactoryExtensions.IsUIVisible(interpreter)) {

```

Comment:
I'd rather have a `using` + call it as instance method for clarity (and the same for the diff above).